diff --git a/Content.Shared/Botany/Systems/PlantSystem.cs b/Content.Shared/Botany/Systems/PlantSystem.cs
index 22ca145ad1..ce04f615a7 100644
--- a/Content.Shared/Botany/Systems/PlantSystem.cs
+++ b/Content.Shared/Botany/Systems/PlantSystem.cs
@@ -163,7 +163,7 @@ public void ForceUpdate(Entity<PlantComponent?> ent)
     public bool TryGetTray(Entity<PlantComponent?> ent, out Entity<PlantTrayComponent> trayEnt)
     {
         trayEnt = default!;
-        if (!Resolve(ent.Owner, ref ent.Comp))
+        if (!Resolve(ent.Owner, ref ent.Comp, false)) // Aavikko fix: don't log error when PlantComponent missing
             return false;
 
         trayEnt.Owner = Transform(ent.Owner).ParentUid;
@@ -346,4 +346,4 @@ public string GetPlantStateMarkup(EntityUid uid, PlantComponent? component = nul
 
         return string.Join("\n", markup);
     }
-}
+}
\ No newline at end of file

