diff --git a/Content.IntegrationTests/Tests/Humanoid/HideableHumanoidLayersTest.cs b/Content.IntegrationTests/Tests/Humanoid/HideableHumanoidLayersTest.cs
index cf20646e48..b59ed038b8 100644
--- a/Content.IntegrationTests/Tests/Humanoid/HideableHumanoidLayersTest.cs
+++ b/Content.IntegrationTests/Tests/Humanoid/HideableHumanoidLayersTest.cs
@@ -59,7 +59,7 @@ await Server.WaitAssertion(() =>
         await Client.WaitAssertion(() =>
         {
             var spriteSystem = CEntMan.System<SpriteSystem>();
-            var snoutIndex = spriteSystem.LayerMapGet(CPlayer, "FoxSnout-vulp_face"); // Corvax-Vulp "VulpSnout-snout"
+            var snoutIndex = spriteSystem.LayerMapGet(CPlayer, "VulpSnout-snout"); // Corvax-Vulp "VulpSnout-snout"
             var snoutCoverIndex = spriteSystem.LayerMapGet(CPlayer, "VulpSnoutNose-snout-nose");
             var spriteComp = CEntMan.GetComponent<SpriteComponent>(CPlayer);
 
@@ -77,7 +77,7 @@ await Server.WaitAssertion(() =>
         await Client.WaitAssertion(() =>
         {
             var spriteSystem = CEntMan.System<SpriteSystem>();
-            var snoutIndex = spriteSystem.LayerMapGet(CPlayer, "FoxSnout-vulp_face"); // Corvax-Vulp "VulpSnout-snout"
+            var snoutIndex = spriteSystem.LayerMapGet(CPlayer, "VulpSnout-snout"); // Corvax-Vulp "VulpSnout-snout"
             var snoutCoverIndex = spriteSystem.LayerMapGet(CPlayer, "VulpSnoutNose-snout-nose");
             var spriteComp = CEntMan.GetComponent<SpriteComponent>(CPlayer);
 

